ci/rpm/scan_daos_node.sh

```diff
@@ -14,6 +14,16 @@ pushd "/var/tmp/${lmd_src}"
 sudo ./install.sh
   sudo ln -s /usr/local/maldetect/ /bin/maldet
```

Comment:
Why is a directory being symlinked to `/bin`?  I can't think of any situation where a directory in `/bin` is useful

Reply:
No idea why that was there.  Can not find the article that said to do it now.  Fixed in the current push.